### ISSUE TITLE
Getting duplicate success/failure messages

### ISSUE THREAD

**Issue by jrmuir** [PeteGoo/tcSlackBuildNotifier#127]
Using TC 10.0.1 and the latest master build of tcSlackBuildNotifier we're getting duplicate notifications.  If I check off include build time one message will include it, but not both.

Misconfig on my part?  Intended?

Example output where "Robot Overlord" is the botname:
```
Robot Overlord BOT [3:28 PM]  
Running - Wordpress :: Wordpress Stack :: wordpress-stack #1.0.12
wordpress-stack
----------------
Commits
----------------
3dcaa9c3a5 :: jrmuir :: I don't need that
Changes By
----------------
jrmuir
[3:28]  
Succeeded - Wordpress :: Wordpress Stack :: wordpress-stack #1.0.12
wordpress-stack
----------------
Commits
----------------
3dcaa9c3a5 :: jrmuir :: I don't need that
Changes By
----------------
jrmuir
[3:28]  
Succeeded - Wordpress :: Wordpress Stack :: wordpress-stack #1.0.12
wordpress-stack
----------------
Commits
----------------
3dcaa9c3a5 :: jrmuir :: I don't need that
Changes By
----------------
jrmuir
```

**Comment by jamesmbourne** [comment]
I'm on TeamCity 10.0.2 with v1.4.6 of the plugin and experiencing the same issue. We're receiving 1 build start notficiation, but then 2 success/failure notifications.

**Comment by timotyweis** [comment]
@jrmuir and @jamesmbourne, could it be that you have the **Build Almost Completed** flag active in the project settings? I experienced the same, but after de-activating this event I'm only receiving one notification. 
Could it be that the notification's title is misleading ("Success" instead of "Almost completed")?

**Comment by jamesmbourne** [comment]
@timotyweis I've just changed that configuration for my projects and this has resolved the issue for me. I now only recieve one notification per build completion. Thanks! 👍

**Comment by pointerxr** [comment]
Thanks @timotyweis ! Much appreciated.

**Comment by jrmuir** [comment]
That's exactly it.  The message isn't indicating almost success, just success.

**Comment by holinov** [comment]
Just same! Turning off "Almost complete" helped.

**Comment by cottsak** [comment]
Same: disabling **Build Almost Completed** made the duplicate go away.

**Comment by PeteGoo** [comment]
Yeah, that almost complete had never been useful to anyone I suspect. Sorry

**Comment by cottsak** [comment]
@PeteGoo Don't be sorry. I've seen heaps of builds that take >10 mins. Some even hours.

These days however, if you need a "almost complete" notification, then you probably should be solving a different problem.

**Comment by xalikoutis** [comment]
Hi guys i am on teamcity 2017 where is this flag? Cant find it....
